README.md

```diff
@@ -8,12 +8,43 @@ when they expire.
 Here a "quote" means an [MCIP #31](https://github.com/mobilecoinfoundation/mcips/31) "Signed Contingent Input",
 which is (informally) a signed offer to trade one cryptocurrency on the Mobilecoin network for another at some price.
 
+See also [MCIP #42](https://github.com/mobilecoinfoundation/mcips/42) which added support for partial fills.
+
 This quoting service can be used to facilitate private peer-to-peer trades.
 
-## License
+## Overview
 
-This code is available under open-source licenses. Look for the [LICENSE](./LICENSE) file for more
-information.
+The DEQS is organized as a decentralized peer-to-peer network for distributing signed quotes.
```

Comment:
It might be worth mentioning our assumptions about the quote structure. Especially the fact that we assume it's between 2 tokens etc.